.make/test.mk

```diff
@@ -117,6 +117,14 @@ ALL_PKGS_EXCLUDE_PATTERN = 'vendor\|app\|tool\/cli\|design\|client\|test'
 GOANALYSIS_PKGS_EXCLUDE_PATTERN="vendor|app|client|tool/cli"
 GOANALYSIS_DIRS=$(shell go list -f {{.Dir}} ./... | grep -v -E $(GOANALYSIS_PKGS_EXCLUDE_PATTERN))
 
+# Folder with contract tests
+CONTRACT_TESTS=$(CUR_DIR)/test/contracts
+
+# Configuration of contract tests
+PACT_VERSION ?= 1.0.0
+PACT_BROKER_URL ?= http://pact-broker-pact-broker.193b.starter-ca-central-1.openshiftapps.com
```

Comment:
We can't relay on OSO-starter to run the broker. Need a better place for that.

Reply:
Maybe the same home as sentry and ike bots? /cc @kbsingh

Reply:
@alexeykazakov @bartoszmajsak Thanks for the suggestion, created appropriate SD ticket  for that (https://gitlab.cee.redhat.com/dtsd/housekeeping/issues/2383)